test/system/710-start.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats   -*- bats -*-
+
+load helpers
+
+function setup() {
+    basic_setup
+    run_podman run -d $IMAGE /bin/true
+    cid_none="$output"
+    run_podman run -d --restart=always $IMAGE /bin/true
+    cid_always="$output"
+}
+
+function teardown() {
+    basic_teardown
+}
+
+@test "podman start --filter - start only containers that match the filter" {
+    run_podman start --filter restart-policy=always $cid_none $cid_always
+    is "$output" "$cid_always"
```

Comment:
As a courtesy to someone reading a test log, could you add a third argument here (and line 22 and 27) with a short description of what this error means? E.g., `"CID of restart-policy=always container"`